facilitation_guide/learning_objectives.rmd

```diff
@@ -1,42 +1,51 @@
-testing branch
+Session 1 - Our Vision 
+CORE OBJECTIVE: To understand Modeling to Learn (MTL) as a way to address a team's needs in meeting VA standards and initiatives using localized patient and provider data.
 
-Session 1 - Our Vision  
 1.	Explain how Modeling to Learn (MTL) seeks to promote team work to improve quality of care.  
 2.	Articulate a team need that MTL will address.  
 3.	Recognize trade offs in quality of work life and quality of care.  
 4.	Describe how MTL uses patient and provider data to support VA standards and initiatives.  
 
 Session 2. Our Data  
+CORE OBJECTIVE: To develop familiarity in accessing localized team data.
+
 1.	Demonstrate how to access and update the team’s patient data on the SharePoint site.  
 2.	Identify a specific patient in the team data.  
 3.	Describe how the team data are categorized into visits by CPT Codes.  
 4.	Specify the team’s preferred data for Modeling to Learn.  
 5.	Describe the limits of retrospective data review for guiding change.  
 
 Session 3. Our System  
+CORE OBJECTIVE: To simulate clinic units (stocks: e.g., appointments available, and flows: e.g., booking rate) in the Care Coordination model in order to improve team decision making.
```

Comment:
The core objective of Session 3 might be to learn the notation and nomenclature of system dynamics diagrams in preparation for using the models. ?